contracts/core/AuxiliaryStake.sol

```diff
@@ -61,6 +61,19 @@ contract AuxiliaryStake {
 
     /* Public Variables */
 
+    /**
+     * The OSTblock gate is the only contract that is allowed to update the
+     * OSTblock height.
+     */
+    address public ostBlockGate;
+
+    /**
+     * Tracks the current height of the OSTblock within this contract. We track
+     * this here to make certain assertions about newly reported OSTblocks and
+     * to know what current height we are voting on.
+     */
+    uint256 public currentOstBlockHeight;
+
     /**
      * Maps auxiliary addresses of validators to their details.
```

Comment:
the validator is a tuple of (origin, auxiliary) key, always referenced by the origin key. can be corrected later

Reply:
The thinking was that we only need the auxiliary addresses on auxiliary.
We should never need the addresses from origin here and the auxiliary addresses are unique.
Right?